### DISCUSSION TITLE
Constraint Icons in N-Panel

### DISCUSSION THREAD

**Discussion by coleorloff** [hlorus/CAD_Sketcher#122] [Feature Requests]
**Motivation / Existing Problem**
It's a little difficult to "read" sketches without familiarity with the constraint icons. (It's also a Blender UI convention to have icons paired with menu items.)

**Description**
Add constraint icons next to their name in the CAD Sketcher N-panel. T

his would provide a positive feedback loop in terms of visually identifying symbols within the system and "reading" your sketches, both for beginners and more advanced users.

**Mock-Up**
Note: highlighted area shows a selection of constraints paired with their icons. The goal would be to have all constraints in the menu paired with their icon. This is only for illustrative purposes. 

![CADSketcher_N-panel_proposal](https://user-images.githubusercontent.com/8648879/167663531-5cf0eb98-793a-4ee2-812e-39c235e36e85.png)

**Comment by SiboVG** [comment]
@hlorus should this be marked as implemented?

**Comment by hlorus** [comment]
Jep you're right!
